client/src/app/pages/login/login.page.html

```diff
@@ -1,145 +1,116 @@
-        </ion-card>
-
-        <form [formGroup]="authForm">
-          <ion-card class="login">
-
-            <ion-card-header>
-              <ion-card-title>
-                {{ titles[currentLoginState] }}
-              </ion-card-title>
-            </ion-card-header>
-
-            <ion-card-content>
-
-              <ion-list>
-                <ion-item>
-                  <ion-label position="stacked">Username (Email)</ion-label>
-                  <ion-input type="email" formControlName="email" (keyup.enter)="handleSubmit()"></ion-input>
-                </ion-item>
-
-                <div class="validation-errors">
-                  <ng-container *ngFor="let validation of validationMessages.email">
-                    <div class="message" *ngIf="authForm.get('email').hasError(validation.type) && (authForm.get('email').dirty || authForm.get('email').touched)">
-                      <ion-icon name="warning-outline"></ion-icon> {{ validation.message }}
-                    </div>
-                  </ng-container>
+    <div class="superior">
+        <div class="contenedor">
+            <div class="top"></div>
+            <center>
+                <div class="logo"></div>
+            </center>
+            <div class="altitud"></div>
+            <div class="contenido">
+                <div class="item1">
+                    <div class="campaing">Campaign Tools For Your Woodland</div>
+                    <div class="texto">Ready to run your own campaign of Root: The Roleplaying Game? The digital tools here will make it a breeze! You can track your own version of the Woodland with its individual clearings, factions, and NPCs, along with the player characters and all their relevant attributes! Just sign-in and get creating!</div>
                 </div>
-
-                <ion-item *ngIf="currentLoginState !== LoginState.Forgot">
-                  <ion-label position="stacked">Password</ion-label>
-                  <ion-input type="password" formControlName="password" (keyup.enter)="handleSubmit()"></ion-input>
-                </ion-item>
-
-                <div class="validation-errors" *ngIf="currentLoginState !== LoginState.Forgot">
-                  <ng-container *ngFor="let validation of validationMessages.password">
-                    <div class="message" *ngIf="authForm.get('password').hasError(validation.type) && (authForm.get('password').dirty || authForm.get('password').touched)">
-                      <ion-icon name="warning-outline"></ion-icon> {{ validation.message }}
-                    </div>
-                  </ng-container>
+                <div class="item2">
+                    Grab a physical copy of the <b>Root: The Roleplaying Game Core Book</b> for everything you need to play! Each copy comes with a free digital version!
+                    <br><br>
+                    <div class="botonamarillo" onclick="location.href='https://magpiegames.com/pages/root-rpg'">buy now</div>
```

Comment:
- please adjust this name to match the others
- don't do onclick. also, dont do this with a div. use a button or something else. ionic has tools for this.